src/iframe/src/canvasAPI/pixel.js

```diff
@@ -0,0 +1,29 @@
+import colors from '../colors.js'
+
+const backgroundColor = 7
+
+const getPixel = ({ x, y, ctx }) => {
+  x = x % 128
```

Comment:
Will this be confusing to users who put in `x = 128`? What will happen if we leave `xy` as they come in?

Reply:
This is a good question. I don't really have an opinion either way, but I see a couple of options.

1. Leave it as is and document the behavior better.
2. Return null or 0 if the coordinates are out of bounds.
3. Throw an error (which should show a message)

I can make another PR with whatever option you prefer. Just let me know.

Reply:
I ended up leaving `xy` as it comes in. Otherwise you have to mess with the camera. It works, but let me know if you see strangeness.

Reply:
Awesome. I didn't even think about camera issues, so this works out for the best.